components/sysutils/gnupg/test/results-64.master

```diff
@@ -0,0 +1,92 @@
+PASS: t-timestuff
+PASS: t-convert
+PASS: t-percent
+PASS: t-gettime
+PASS: t-sysutils
+PASS: t-sexputil
+PASS: t-session-env
+PASS: t-openpgp-oid
+PASS: t-ssh-utils
+PASS: t-mapstrings
+PASS: t-zb32
+PASS: t-mbox-util
+PASS: t-iobuf
+PASS: t-strlist
+PASS: t-name-value
+PASS: t-ccparray
+PASS: t-recsel
+PASS: t-exechelp
+PASS: t-exectool
+PASS: t-rmd160
+PASS: t-keydb
+PASS: t-keydb-get-keyblock
+PASS: t-stutter
+PASS: t-http-basic
+PASS: t-ldap-parse-uri
+PASS: tests/openpgp/setup.scm 
+PASS: tests/openpgp/version.scm 
+PASS: tests/openpgp/enarmor.scm 
+PASS: tests/openpgp/mds.scm 
+PASS: tests/openpgp/decrypt.scm 
+PASS: tests/openpgp/decrypt-multifile.scm 
+PASS: tests/openpgp/decrypt-dsa.scm 
+PASS: tests/openpgp/decrypt-session-key.scm 
+PASS: tests/openpgp/decrypt-unwrap-verify.scm 
+PASS: tests/openpgp/sigs.scm 
+PASS: tests/openpgp/sigs-dsa.scm 
+PASS: tests/openpgp/encrypt.scm 
+PASS: tests/openpgp/encrypt-multifile.scm 
+PASS: tests/openpgp/encrypt-dsa.scm 
+PASS: tests/openpgp/compression.scm 
+PASS: tests/openpgp/seat.scm 
+PASS: tests/openpgp/clearsig.scm 
+PASS: tests/openpgp/encryptp.scm 
+PASS: tests/openpgp/detach.scm 
+PASS: tests/openpgp/detachm.scm 
+PASS: tests/openpgp/armsigs.scm 
+PASS: tests/openpgp/armencrypt.scm 
+PASS: tests/openpgp/armencryptp.scm 
+PASS: tests/openpgp/signencrypt.scm 
+PASS: tests/openpgp/signencrypt-dsa.scm 
+PASS: tests/openpgp/armsignencrypt.scm 
+PASS: tests/openpgp/armdetach.scm 
+PASS: tests/openpgp/armdetachm.scm 
+PASS: tests/openpgp/genkey1024.scm 
+PASS: tests/openpgp/conventional.scm 
+PASS: tests/openpgp/conventional-mdc.scm 
+PASS: tests/openpgp/multisig.scm 
+PASS: tests/openpgp/verify.scm 
+PASS: tests/openpgp/verify-multifile.scm 
+PASS: tests/openpgp/gpgv.scm 
+PASS: tests/openpgp/gpgv-forged-keyring.scm 
+PASS: tests/openpgp/armor.scm 
+PASS: tests/openpgp/import.scm 
+PASS: tests/openpgp/import-revocation-certificate.scm 
+PASS: tests/openpgp/4gb-packet.scm 
+PASS: tests/openpgp/tofu.scm 
+PASS: tests/openpgp/trust-pgp-1.scm 
+PASS: tests/openpgp/trust-pgp-2.scm 
+PASS: tests/openpgp/trust-pgp-3.scm 
+PASS: tests/openpgp/use-exact-key.scm 
+PASS: tests/openpgp/default-key.scm 
+PASS: tests/openpgp/ssh-export.scm 
+PASS: tests/openpgp/key-selection.scm 
+PASS: tests/openpgp/delete-keys.scm 
+PASS: tests/openpgp/gpgconf.scm 
+PASS: tests/openpgp/issue2015.scm 
+PASS: tests/openpgp/issue2346.scm 
+PASS: tests/openpgp/issue2417.scm 
+PASS: tests/openpgp/issue2419.scm 
+PASS: tests/openpgp/issue2941.scm 
+PASS: tests/migrations/from-classic.scm 
+PASS: tests/migrations/extended-pkf.scm 
+PASS: tests/migrations/issue2276.scm 
+PASS: tests/gpgsm/setup.scm 
+PASS: tests/gpgsm/import.scm 
+PASS: tests/gpgsm/encrypt.scm 
+PASS: tests/gpgsm/verify.scm 
+PASS: tests/gpgsm/decrypt.scm 
+PASS: tests/gpgsm/sign.scm 
+PASS: tests/gpgsm/export.scm 
+SKIP: Environment variable 'XTEST_GPGME_SRCDIR' not set.  Please point it to a recent GPGME source tree to run the GPGME test suite. 
```

Comment:
The message is clear: If you want to let this test being run, you'll need to set the XTEST_GPGME_SRCDIR variable to refer to the GPGME sources, eg. add the following line to the Makefile:
COMPONENT_TEST_ENV += $(COMPONENT_DIR)/../gpgme/gpgme-1.13.1
I am not sure whether this works, though. You maybe need to experiment with the settings.

Reply:
Thanks, I'll try it and play with it.

Reply:
Ups, should be
COMPONENT_TEST_ENV += XTEST_GPGME_SRCDIR=$(COMPONENT_DIR)/../gpgme/gpgme-1.13.1

Reply:
But IMO it's not that important to let this test case succeed. Skipping is ok for me.

Reply:
Pro-tip: look for similar things in our sources, eg.
cd .../components
find . -name Makefile | xargs grep COMPONENT_TEST_ENV
Looking for similar solutions is always a good idea ;-)

Reply:
> Ups, should be
> COMPONENT_TEST_ENV += XTEST_GPGME_SRCDIR=$(COMPONENT_DIR)/../gpgme/gpgme-1.13.1

Thank you for showing me how to assign two variables at the same time! Yet, it didn't work, now the message says: 
```
SKIP: Environment variable 'XTEST_GPGME_BUILDDIR' not set.  Please point it to a recent GPGME build tree to run the GPGME test suite.
```
It looks like it could go on for a long time.

Reply:
> But IMO it's not that important to let this test case succeed. Skipping is ok for me.

Perfect, then I think it's ready.

Reply:
> Pro-tip: look for similar things in our sources, eg.
> cd .../components
> find . -name Makefile | xargs grep COMPONENT_TEST_ENV
> Looking for similar solutions is always a good idea ;-)


Thank you, normally I would do that (this is how I got COMPONENT_TEST_TRANSFORMS :) but in this case I didn't know what was wrong. And I didn't notice that gpgme is another component...

Reply:
Ok, new trick: run
gmake print-COMPONENT_TEST_ENV
and see what it spits out :)

Reply:
Plus, you may need to run
gmake unpack
for the gpgme component in order make this work.